samples/history/basic.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "1.1.0",
+  "startDate": "2018-10-06T04:00:00Z",
+  "endDate": "2018-10-06T04:00:02Z",
+  "generator": "Hand-written with love with the spec",
+
+  "objects": [
```

Comment:
How about `results`?

Reply:
I don't think this format is suitable for log files, as it is not appendable. Maybe this is just terminology: this works for capturing a set of data in a self contained file and response is not the best choice in that case.

I'd go with something like `data` or `seriesdata`.